test/modules/Modules.Test/TestResultCoordinator/Reports/DirectMethod/DirectMethodConnectivityReportDataWithSenderAndReceiverSource.cs

```diff
@@ -90,7 +108,7 @@ class DirectMethodConnectivityReportDataWithSenderAndReceiverSource
                     // NetworkOnFailure test
                     Enumerable.Range(1, 7).Select(v => (ulong)v),
                     new[] { 1UL, 2UL, 3UL, 5UL, 6UL, 7UL },
-                    new List<HttpStatusCode> { HttpStatusCode.InternalServerError, HttpStatusCode.OK, HttpStatusCode.OK, HttpStatusCode.NotFound, HttpStatusCode.OK, HttpStatusCode.OK, HttpStatusCode.OK },
+                    new List<HttpStatusCode> { HttpStatusCode.OK, HttpStatusCode.OK, HttpStatusCode.InternalServerError, HttpStatusCode.NotFound, HttpStatusCode.InternalServerError, HttpStatusCode.InternalServerError, HttpStatusCode.InternalServerError },
```

Comment:
I wanted to keep the test functioning as expected, ignoring any tolerances. Therefore I had to exceed the tolerance (by adding more internal server errors) so that it would surpass the new threshold. I didn't want to add tests for the tolerances as it gets tricky, we are on a time crunch, they are subject to change, and it is simple percentage logic.

Reply:
I am saying we don't need to test these cases as it is unecessary.